blist.js

```diff
@@ -1,170 +1,176 @@
+            botId = this.client.user.id;
+        };
+
+        let data;
+        await axios.get(`${url}/api/bot/${botId}/votes/`, { headers: { 'Authorization': `${this.apiKey}` } }).then((res) => {
+            data = res["data"];
+        }).catch((error) => {
+            if (error.response.status == 403) {
+                return console.error(`API key does not match bot API key.`);
+            } else if (error.response.status == 404) {
+                return console.error(`A bot with ID: ${id} could not be found on Blist`);
             } else {
-                return console.error(`Error while sending request: ${e.response.status}  `)
-            }
-        })
-        return o
-    }
+                return console.error(`There was an issue while sending a request: ${error}`);
+            };
+        });
+
+        return data;
+    };
 
     async postStats() {
-        var shards;
+        if (!this.apiKey) {
+            return console.log("An API key is required to use this method");
+        };
+
+        let shards;
         try {
-            shards = this.client.shard.fetchClientValues('guilds.size').size
-        } catch(e) {
-            shards = 1
-        }
-        await axios.post(`${url}/api/bot/${this.client.user.id}/stats/`, {server_count:this.client.guilds.cache.size,shard_count:shards}, {headers:{'Authorization':`${this.key}`}}).then((res) => {
-            return "Succesfully posted stats."
-        }).catch((e) => {
-            if (e.response.status == 403) {
-                return console.error(`Client did not provide API key or API key is invalid.`)
-            } else if (e.response.status == 404) {
-                return console.error(`This bot is not on Blist.`)
+            shards = this.client.shard.fetchClientValues('guilds.size').size;
+        } catch (error) {
+            shards = 0;
+        };
+
+        await axios.post(`${url}/api/bot/${this.client.user.id}/stats/`, { server_count: this.client.guilds.cache.size, shard_count: shards }, { headers: { 'Authorization': `${this.apiKey}` } }).then((res) => {
+            return console.log("Succesfully posted stats");
+        }).catch((error) => {
+            if (error.response.status == 403) {
+                return console.error(`API key does not match bot API key.`);
+            } else if (error.response.status == 404) {
+                return console.error(`A bot with ID: ${id} could not be found on Blist`);
             } else {
-                return console.error(`Error while sending request: ${e.response.status}`)
-            }
-        })
-    }
+                return console.error(`There was an issue while sending a request: ${error}`);
+            };
+        });
+    };
 
     async startAutopost(interval) {
         if (this.autopost) {
-            return console.log("Autpost already running.")
-        }
-        var i;
+            return console.log("Autpost is already running")
```

Comment:
```suggestion
            return console.log("Autopost is already running")
```
Look into using `error` or `warn` for this instead.